### DISCUSSION TITLE
Polygon Select

### DISCUSSION THREAD

**Discussion by enderpuff** [PintaProject/Pinta#610] [Ideas]
Pretty self explanatory, but polygon selects are very very useful for editing images, for stuff like isolating objects or removing backgrounds, etc...

**Comment by cameronwhite** [comment]
There is already the Lasso Select tool, so I assume you mean something more for drawing a polygon with perfectly straight edges?

**Comment by acevenoms** [comment]
I'd like to +1 this, yes the lasso select is good for rough selections where you have room around the edge but a polygon select with perfectly straight lines help with precision

**Comment by Acyone** [comment]
It would be nice, instead of click-dragging with the lasso tool, to be able to click the next point and to have the selection draw a line between the last point and the new clicked point. Maybe using shift+click or ctrl + click : This way no new tool would be needed. It is what I intuitively tried at first
